### ISSUE TITLE
`this.throttleTask` doesn't allow extra arguments

### ISSUE THREAD

**Issue by mdebbar** [ember-lifeline/ember-lifeline#41]
What's the best way to convert `Ember.run.throttle('myMethod', arg1, arg2, 100)` to ember-lifeline's alternative?

`this.throttleTask` expects the first argument to be the method name, so we can't pass a curried function or an arrow function. And it doesn't accept extra arguments like its `Ember.run` counterpart.

cc @scalvert @rwjblue

**Comment by billdami** [comment]
It would be nice to have this for the other `RunMixin` methods as well, so they have a signature closer to their `Ember.run` counterparts. E.g.

```js
this.runTask('someMethod', extraArg1, extraArg2, ..., timeout);
```

**Comment by scalvert** [comment]
Resolved in #292
